easydigitaldownloads/themedd
# Set explicit versions for devDependencies.

This PR sets explicit version for devDependencies. It still leaves a couple vulnerabilities, but I couldn't seem to get them fixed. I don't think they matter for actual deployment though.

To test:

- Nuke `node_modules/`
- Run `npm install`
- Ensure `grunt` command still works.
